.github/workflows/os-increment-plugin-versions.yml

```diff
@@ -85,6 +85,36 @@ jobs:
           app_id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.APP_PRIVATE_KEY }}
           installation_id: 22958780
+      - name: Check if label exists
+        id: check_label
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ steps.github_app_token.outputs.token }}
+          result-encoding: string
+          script: |
+            const { data: labels } = await github.rest.issues.listLabelsForRepo({
+              owner: context.repo.owner,
+              repo: "${{ matrix.entry.repo }}",
+              per_page: 100
+            });
+            const labelFound = labels.some(label => label.name === 'v${{ env.OPENSEARCH_VERSION_NUMBER }}');
+            console.log(labelFound);
+            return labelFound
```

Comment:
Interesting, so this acts as a function in github actions?

Reply:
Yes Sayali Once works I will update the document
Also ya this is equivalent to `set-output` https://github.com/actions/github-script#reading-step-results